Makefile

```diff
@@ -211,8 +211,7 @@ release-artifacts: ## Build release artifacts
 ## --------------------------------------
 
 .PHONY: binaries-dev
-binaries-dev: ## Builds and installs all development binaries using go get
-	go get -v ./...
+binaries-dev: generate manager clusterawsadm clusterctl
```

Comment:
this installs the binaries to GOPATH?

Reply:
Yes: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/970144de689f92f1b7d42c34da87b218f1f01dc4/Makefile#L178-L191